src/components/ec-radio-btn/ec-radio-btn.spec.ts

```diff
@@ -0,0 +1,301 @@
+import type { MountingOptions } from '@vue/test-utils';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from '@vue/test-utils';
+
+import type { CVueWrapper } from '../../../tests/utils/global';
+import { EcRadioBtn } from '../../main';
+import type { RadioButtonProps } from './types';
+
+function mountRadioBtn(props?: Partial<RadioButtonProps>, mountOpts?: MountingOptions<RadioButtonProps>): CVueWrapper {
+  return mount(
+    EcRadioBtn as any, // eslint-disable-line
+    {
+      props: {
+        value: 'y',
+        label: 'Yes',
+        ...props,
+      },
+      ...mountOpts,
+    },
+  ) as CVueWrapper;
+}
+
+describe('EcRadioBtn', () => {
+  it('should render properly', () => {
+    const wrapper = mountRadioBtn();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  describe('props', () => {
+    describe('name', () => {
+      it('should have a "name" attribute on the input equal to the "for" attribute on the label', () => {
```

Comment:
`for` attributes of the label tag targets to the `element_id` not for the `name` attribute